src/components/new-safe/create/logic/index.ts

```diff
@@ -86,9 +86,18 @@ export const createNewSafe = async (
 export const computeNewSafeAddress = async (
   ethersProvider: BrowserProvider,
   props: DeploySafeProps,
+  chainId: string,
 ): Promise<string> => {
-  const safeFactory = await getSafeFactory(ethersProvider)
-  return safeFactory.predictSafeAddress(props.safeAccountConfig, props.saltNonce)
+  const ethAdapter = await createEthersAdapter(ethersProvider)
+
+  return predictSafeAddress({
+    ethAdapter,
+    chainId: BigInt(chainId),
+    safeAccountConfig: props.safeAccountConfig,
+    safeDeploymentConfig: {
+      saltNonce: props.saltNonce,
```

Comment:
If we do not pass in the `safeVersion` here the SDK will use its current `DEFAULT_SAFE_VERSION`.
But we store our `LATEST_SAFE_VERSION` in the local storage. 
Updating the protocol-kit could or our interface's safe version could lead to a mismatch  which would lead to a mismatch of the predicted Safe address.

Reply:
We call this function before a safe is created where we use `LATEST_SAFE_VERSION` and don't read anything from local storage. Still makes sense to explicitely add `LATEST_SAFE_VERSION` here in case there is some unexpected change on the protocol-kit side.